src/types/DocumentTypes.ts

```diff
@@ -28,31 +28,31 @@ export interface ChiliDocument {
     properties?: TemplateDocumentProperties | ProjectDocumentProperties;
     pages: DocumentPage[];
     layouts: (ChildLayout | TopLayout)[];
-    styleKit: DocumentStyleKit;
+    stylekit: DocumentStylekit;
     variables: Variable[];
     actions: DocumentAction[];
 }
 
 export interface DocumentPage {
-    pageId: string;
-    pageNumber: number;
+    id: string;
+    number: number;
     frames: (ImageFrame | TextFrame | ShapeFrame)[];
 }
 
 export interface DocumentFrame {
-    frameId: string;
-    frameName: string;
-    frameType: FrameTypeEnum;
+    id: string;
+    name: string;
+    type: FrameTypeEnum;
     blendMode: BlendMode;
     constrainProportions: boolean;
 }
 
 export interface ImageFrame extends DocumentFrame {
-    src: VariableImageFrameSource | AssetProviderImageFrameSource | UrlImageFrameSource;
+    src: VariableImageFrameSource | ConnectorImageFrameSource | UrlImageFrameSource;
```

Comment:
Not commenting the connector change but the three sources naming is inconsistent with `FrameType.ts`.

`export type ImageFrameSource = ImageFrameUrlSource | ImageFrameConnectorSource | ImageFrameVariableSource;`
If team is ok for changing this I personally prefer the `FrameType.ts` version.